cloud_build/deploy.yaml

```diff
@@ -0,0 +1,19 @@
+steps:
+  - name: gcr.io/cloud-builders/git
+    args: ['submodule', 'update', '--init', '--recursive']
+  - name: gcr.io/cloud-builders/docker
+    entrypoint: '/bin/bash'
+    args:
+      - '-c'
+      - |-
+        set -e
+        echo "Building the website using a makefile..."
+        make build
+  - name: gcr.io/flutter-dev-230821/firebase-ghcli
+    entrypoint: '/bin/bash'
+    args:
+      - '-c'
+      - |-
+        firebase deploy --project=flutter-website-staging-349021 --only=hosting
```

Comment:
Is this supposed to be to a dart project instead?

Reply:
Yes! I'm going to open another PR to change that once this lands and we've confirmed it deploys properly. I don't want to remove the current deploy GitHub action until we confirm this deploy works & I don't want them to deploy on top of each other.